src/services/auth.js

```diff
@@ -14,6 +14,7 @@ async function login(data) {
     asyncStore.save("user", JSON.stringify(credentials));
     return credentials;
   } catch (err) {
+    console.log("auth login POST error: ", err.response || err);
```

Comment:
We might consider integrating the App Center error tracking service, or some similar service, soon so that we can see errors like this on real devices.